### ISSUE TITLE
Allow setting light/dark mode

### ISSUE THREAD

**Issue by turbanisch** [sbarex/QLMarkdown#131]
I have dark mode enabled in my system settings (on MacOS) yet I prefer the Markdown preview in light mode. Is there a setting that allows this? If not, I would love to have one. I have seen the light/dark mode toggle in the lower right corner of the settings but this seems to affect only the settings window, not the actual preview?

As a workaround, I have currently downloaded the default theme and removed the parts that change the CSS with `prefers-color-scheme`.

**Comment by sbarex** [comment]
No, for now there is not settings to force a fixed light/dark theme.

**Comment by laaggii** [comment]
If someone, like OP or myself, is looking for steps to force switch to light theme on MacOS, here you go:

- Go to /Applications/QLMarkdown.app/Contents/Resources/default.css
- Remove "@media (prefers-color-scheme: dark) {....} block
- In terminal

```
qlmanage -r
qlmanage -r cache
killall Finder
```

**Comment by sbarex** [comment]
`qlmanage` do not handle this QuickLook appex.

**Comment by soreavis** [comment]
@sbarex — this is the "force a fixed light/dark theme" request from this issue. I had a go at it and have a working prototype; sharing here before opening any PR, in case the approach is useful or you'd want it shaped differently.

**What it does** — adds an **Appearance** control to Preferences (Auto / Light / Dark):

- **Auto** — follows macOS exactly as today. Default, so nothing changes for existing users.
- **Light / Dark** — forces the rendered Markdown to that appearance regardless of the system setting, in **both the in-app preview and the Quick Look output**. It's essentially the automated version of the manual "remove the `@media (prefers-color-scheme: dark)` block from `default.css`" workaround mentioned in this thread: when a fixed mode is chosen it just rewrites that media condition (`not all` / `all`) so the bundled CSS resolves to the chosen scheme — no CSS parsing, the default theme is untouched on disk.
- The chosen mode also themes the **settings window chrome**, so it folds in the old bottom-bar light/dark toggle (which only affected the Preferences window). One setting now drives preview + Quick Look + window.

It's persisted like every other setting (added to the `Settings` model, shared via the App Group), so the Quick Look extension honours it across processes.

**Settings window**

| Settings Light | Settings Dark |
|---|---|
| ![settings-light](https://s.soreavis.com/SCR-20260629-gcdp.png) | ![settings-dark](https://s.soreavis.com/SCR-20260629-g8wp.png) |

**Quick Look preview** (macOS in **Dark** mode in both shots, so the light one is the forced override):

| Forced Light | Forced Dark |
|---|---|
| ![preview-light](https://s.soreavis.com/SCR-20260629-g91p.png) | ![preview-dark](https://s.soreavis.com/SCR-20260629-g97p.png) |

Verified on the in-app preview, the CLI, the real Finder spacebar Quick Look (cross-process), and a save → quit → relaunch round-trip.

The prototype lives on [`feature/preview-appearance-mode`](https://github.com/soreavis/QLMarkdown/tree/feature/preview-appearance-mode) if you'd like to look. Happy to open a PR if you're interested. I've held it pending your take on the approach — in particular whether folding the existing settings-window light/dark toggle into this one Appearance control is the direction you'd want, or whether you'd prefer to keep them separate. Just say the word and I'll polish it up and send it.

**Comment by sbarex** [comment]
Implemented on release 1.5.2.
